python/cudf/cudf/core/udf/strings_typing.py

```diff
@@ -0,0 +1,236 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+
+import operator
+
+from numba import types
+from numba.core.typing import signature as nb_signature
+from numba.core.typing.templates import AbstractTemplate, AttributeTemplate
+from numba.cuda.cudadecl import registry as cuda_decl_registry
+
+from strings_udf._typing import StringView, string_view
+
+from cudf.core.udf import masked_typing
+from cudf.core.udf._ops import comparison_ops
+from cudf.core.udf.masked_typing import MaskedType
+
+masked_typing.MASKED_INIT_MAP[types.pyobject] = string_view
+masked_typing.MASKED_INIT_MAP[string_view] = string_view
+
+
+def _is_valid_string_arg(ty):
+    return (
+        isinstance(ty, MaskedType) and isinstance(ty.value_type, StringView)
+    ) or isinstance(ty, types.StringLiteral)
+
+
+# String functions
+@cuda_decl_registry.register_global(len)
+class MaskedStringViewLength(AbstractTemplate):
+    """
+    provide the length of a cudf::string_view like struct
+    """
+
+    def generic(self, args, kws):
```

Comment:
This function does nothing. It returns `None` regardless of the args passed. Just write

```python
def generic(self, args, kws):
    return None  # or pass
```